src/helpers/shared/user-login.js

```diff
@@ -9,6 +9,11 @@ import { AccessApi, PrincipalApi, GroupApi } from '@redhat-cloud-services/rbac-c
 
 const axiosInstance = axios.create();
 
+const paramSerializer = config => {
+  config.url = config.url.replace(/(?==)*%+/g, value => value.replace(/%/g, '%25%0A'));
```

Comment:
Why the %0A?   That's a newline character, and should not be needed.

Additionally, I'm concerned this will replace any %encoded values already in there... for example, if `%25` is already in the string, this will make it `%25%0A25` (or without the `0A` part, just `%2525`).  Is there a reason you're not using some off-the-shelf params encoder instead of rolling your own (or was that just for expediency)?

Reply:
`%0A` that is a mistake on my part. Honestly I don't know why how it go there...

This is just a hot fix. I want to add some proper encoding to all values not just `%`. It will be in the JS client package.